test/benchmarks_crhmc.cpp

```diff
@@ -0,0 +1,81 @@
+// VolEsti (volume computation and sampling library)
```

Comment:
is there any way you can put assert statements here?

Reply:
Done there are now assert statements depending on the completion time.